pkgs/by-name/fe/fex/package.nix

```diff
@@ -50,6 +50,7 @@ llvmPackages.stdenv.mkDerivation (finalAttrs: rec {
     "-DUSE_LINKER=lld"
     "-DENABLE_LTO=True"
     "-DENABLE_ASSERTIONS=False"
+    (lib.cmakeFeature "OVERRIDE_VERSION" finalAttrs.version)
```

Comment:
You should also remove the `gitMinimal` dependency, as my original commit did: https://github.com/NixOS/nixpkgs/pull/374694/commits/011e3cfea9c900c4074de79b9ea0cfed626c6352